packages/nodes-base/nodes/Slack/Slack.node.ts

```diff
@@ -276,6 +280,40 @@ export class Slack implements INodeType {
 				return returnData;
 			},
 		},
+		credentialTest: {
+			async testSlackTokenAuth(this: ICredentialTestFunctions, credential: ICredentialsDecrypted): Promise<NodeCredentialTestResult> {
+
+				const options = {
+					method: 'POST',
+					headers: {
+						'Content-Type': 'application/json; charset=utf-8',
+						Authorization: `Bearer ${credential.data!.accessToken}`,
+					},
+					uri: 'https://slack.com/api/users.profile.get',
+					json: true,
+				};
+
+				try {
+					const response = await this.helpers.request(options);
+					if (!response.ok) {
+						return {
+							status: 'Error',
+							message: `Unable to authenticate: ${response.error}`,
```

Comment:
Please remove `Unable to authenticate`.. it's redundant with the title of the error box in the designs...